http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/config/Config.scala

```diff
@@ -607,6 +607,10 @@ object Config {
   implicit private val galaxyDrsUrlReader: ValueReader[GalaxyDrsUrl] = stringValueReader.map(GalaxyDrsUrl)
 
   implicit private val galaxyAppConfigReader: ValueReader[GalaxyAppConfig] = ValueReader.relative { config =>
+    val drsUrl = config.as[String]("drsResolver") match {
+      case "martha" => config.as[GalaxyDrsUrl]("marthaUrl")
+      case "drshub" => config.as[GalaxyDrsUrl]("drshubUrl")
```

Comment:
good question. @tlangs do you know the answer to this?

Reply:
the ticket mentions "Just configuration options that aren't utilized yet"

Reply:
Yeah. We haven't performance-tested DRSHub yet, and making things talk to it without first doing that seems unwise. The plan is to test it sequentially in all pre-prod envs to make sure nothing breaks, then performance test it  to make sure it can at least match Martha, then push to prod.

Reply:
Yeah. We want to make sure DRSHub can match Martha's performance before going to prod. The plan is to configure all services to be able to talk to DRSHub, then flip the switches in non-prod envs sequentially and see what happens. We'll want to perf test it before going to prod as well. So for now, things should continue using Martha.

Reply:
I probably would've just used different value for different envs instead of introducing 2 config values.

So basically just adding `drshubUrl`, and the value for it will vary depending on the env, which will be a firecloud-develop PR

Reply:
So, I'm hearing you'd rather have the rendered environment determine the `drsUrl`, rather than doing it in code?

Reply:
Going with this approach, I'm realizing that the unit tests can either test Martha or DRSHub but not both.  I'm wondering if we should leave the Martha code or if we want to switch over tests to DRSHub now. I think could keep the Martha code until we're ready to release the DRSHub code. No strong preference.

Reply:
for just tests, just use whatever that's in reference.conf

Reply:
just to clarify, you're saying we should keep the old values (referencing Martha), correct?

If so, I'll leave this PR open until the cutover to DRSHub.

Reply:
I'm saying for tests, you can just update it to whatever value you change to in `reference.conf`...like here in this PR, you updated to drs url, so just use that in test too...

The real value is controlled by config value in firecloud-develop, so whether or not we want to merge this PR is up-to-you...doesn't really matter whether we merge now or later as long as `firecloud-develop` PR is merged

Reply:
oh gotcha, thanks for clarifying! I'll go ahead and merge this then to reduce code hanging around